modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -17,8 +17,9 @@ use Drupal\user\Entity\Role;
  * @throws Exception
  */
 function acquia_cms_site_studio_install() {
-  $site_studio_api_key = getenv('SITESTUDIO_API_KEY');
-  $site_studio_org_key = getenv('SITESTUDIO_ORG_KEY');
+  $config = Drupal::config('cohesion.settings');
+  $site_studio_api_key = $config->get('api_key');
```

Comment:
Can we update this to pick variable from env and settings override both:
$site_studio_api_key = getenv('SITESTUDIO_API_KEY') ?? $config->get('api_key');
$site_studio_org_key = getenv('SITESTUDIO_ORG_KEY') ?? $config->get('api_key');
if key available in env then pick else look for settings override.